### PR TITLE
id res bq

### PR DESCRIPTION
**Pull request by gane5hvarma**
**Fixes** # (*issue*)

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests for the code
- [ ] I have made corresponding changes to the documentation

---
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-server/885)

### CODE REVIEW

File: warehouse/bigquery/bigquery.go [hunk 1]
```diff
@@ -462,21 +495,212 @@ func (bq *HandleT) Cleanup() {
...
+	}
+
+	if status.Err() != nil {
+		return status.Err()
+	}
 	return
 }
 
 func (bq *HandleT) LoadIdentityMappingsTable() (err error) {
+	identityMappingsTable := warehouseutils.IdentityMappingsWarehouseTableName(PROVIDER)
```
Comment (psrikanth88) [inline warehouse/bigquery/bigquery.go:538]: merge both LoadIdentityMergeRulesTable and LoadIdentityMappingsTable into one func. Or maybe reuse the loadTable func as we are not doing anything different with dedup right?

**Review by psrikanth88 — CHANGES_REQUESTED** [review]
@gane5hvarma Also while creating view for identity mappings table we should partition by merge_property_type + merge_property_value and return the latest rudderId for the combination. Deduping on id in view will not be useful

**Review by psrikanth88 — APPROVED** [review]

**Review by sivashanmukh — APPROVED** [review]
LGTM

**Review by sivashanmukh — APPROVED** [review]
LGTM

**Pull request merged**